### PR TITLE
Node2D/Control variables now update with canvas changes

### PR DESCRIPTION
**Pull request by robertdhernandez**

### CODE REVIEW

**Comment by bojidar-bg** [comment]
@robertdhernandez Would you mind squashing the two commits together (feel free to drop by on IRC and ask for help)

**Comment by robertdhernandez** [comment]
@bojidar-bg Done

File: scene/2d/node_2d.cpp [hunk 1]
```diff
@@ -95,15 +95,15 @@ void Node2D::edit_set_rect(const Rect2 &p_edit_rect) {
...
+	_change_notify("scale");
+	_change_notify("position");
 }
 
 void Node2D::edit_rotate(float p_rot) {
 
 	angle += p_rot;
 	_update_transform();
-	_change_notify("transform/rot");
+	_change_notify("rotation");
```
Comment (bojidar-bg) [inline scene/2d/node_2d.cpp]: Shouldn't you add rotation_deg here as below?

File: scene/2d/node_2d.cpp [hunk 2]
```diff
@@ -63,9 +63,9 @@ void Node2D::edit_set_state(const Variant &p_state) {
 	angle = state[1];
 	_scale = state[2];
 	_update_transform();
-	_change_notify("transform/rot");
-	_change_notify("transform/scale");
-	_change_notify("transform/pos");
+	_change_notify("rotation");
```
Comment (bojidar-bg) [inline scene/2d/node_2d.cpp]: Shouldn't you add rotation_deg here as below? (L146)

**Review by bojidar-bg — APPROVED** [review]
Otherwise, LGTM

**Comment by robertdhernandez** [comment]
Thanks for catching that. I've changed that now.

**Review by 27thLiz — APPROVED** [review]

**Pull request merged**